validator/Cargo.toml

```diff
@@ -18,7 +18,10 @@ futures = "0.3.1"
 iron = "0.6.1"
 log = "0.4"
 pretty_env_logger = "0.3"
+router = "0.6.0"
 serde = "1.0.104"
+serde_derive = "1.0.104"
```

Comment:
We definitely don't need to import `serde_derive` as seperate dependency as everything is already included in base `serde` (as long as we use correct features). I think the same might be  true for `serde_json` but I haven't tested that claim

Reply:
Good catch on the `dervive`. It looks the `serde_json` is needed.